gradle/gradle-mvn-push.gradle

```diff
@@ -21,7 +21,7 @@ version = VERSION_NAME
 group = GROUP
 
 def isReleaseBuild() {
-  return VERSION_NAME.contains("SNAPSHOT") == false
+  return !isCi && VERSION_NAME.contains("SNAPSHOT") == false
```

Comment:
Ah, I never hit this because I never pushed a change to master with a VERSION_NAME that didn't contain SNAPSHOT. My approach was to simply bump the version for the purpose of the release on a branch, tag that commit and push the tags, and then submit a PR with new version plus '-SNAPSHOT' appended. I have no idea which approach is preferred, but either will work.

Reply:
cc @swankjesse for his thoughts since I cribbed the release notes from SQLDelight.

Reply:
Yeah, that's fine too, this is a nice safety check, I think.